cross-media-measurement/src/main/kotlin/org/wfanet/measurement/duchy/db/computation/testing/FakeComputationsDatabaeTransactor.kt

```diff
@@ -41,7 +42,7 @@ private const val PRIMARY_WORKER = "PRIMARY_WORKER"
 /**
  * In-memory [ComputationsDatabase]
  */
-class FakeComputationDb private constructor(
+class FakeComputationsDatabaeTransactor private constructor(
```

Comment:
this is not just a transactor I think.

FakeComputationsDatabase or FakeComputationsDd seem better to me.

Reply:
and it is missing 's' in "database"

Reply:
done